### DISCUSSION TITLE
Nuxt 3 prerequisities - possible docs enhancement?

### DISCUSSION THREAD

**Discussion by AloisSeckar** [nuxt/nuxt#18239] [Ideas]
Hi. I started with Nuxt 3 on nearly brand new Windows 10 computer with almost nothing prepared.

Checked https://v3.nuxtjs.org/getting-started/introduction/ for requirements, then moved to https://v3.nuxtjs.org/getting-started/installation to try: 

`npx nuxi init nuxt3-app`

Encountered two problems though:

1. no **git** installed (although I already had operational _GitHub desktop_), so I had to install _Git For Windows_ first
2. then I ran into Windows security settings for PowerShell. The default policy for executing scripts on my computer was `Restricted`, so I had to run PowerShell command `Set-ExecutionPolicy -ExecutionPolicy Unrestricted` before I was allowed to run the installation command

After solving these two, I created starter app smoothly.

So I humbly suggest considering enhancing "Getting started" docs with those two posibilites. Hope I am not the only guy trying to use Windows for website development :D

**Comment by kissu** [comment]
This is some common workflow setup to start working on Windows and not directly related to Nuxt IMO.
